client/src/constants/backend_constants.js

```diff
@@ -36,6 +36,48 @@ const BACKEND_CONSTANTS = {
   tracer_profile_model: null,
   tracer_concentration_model: null,
   tracer_density: null,
+  cosmo_params: {
+    Planck13: {
+      H0: 67.77,
+      Ob0: 0.048252,
+      Om0: 0.30712,
+      z: 0.0,
+      n: 0.9619,
+      sigma_8: 0.8347,
+    },
```

Comment:
Nice to see these! One thing you could do that might help simplify a lot of your initialization & changed option logic is keeping these structured like you expect them to be in the payload.

If you have:
{Planck13: {
      z: 0.0,
      n: 0.9619,
      sigma_8: 0.8347,
      cosmo_params: {
        H0: 67.77,
        Ob0: 0.048252,
        Om0: 0.30712,
      }, ...
}

a lot of the manual work you do currently is just a spread operator away!

Not a big deal if you don't decide to change -- just a suggestion to make it a little cleaner.

Reply:
Updated to match this.